### DISCUSSION TITLE
laser 30 Watt

### DISCUSSION THREAD

**Discussion by wiwan1971** [arkypita/LaserGRBL#1162] [Q&A]
Ciao a tutti, ho un laser da 30 watt nel database materiali non ce un laser 30 watt ma solo fino a 20 come posso aggiungerlo per dargli più potenza, grazie

**Comment by arkypita** [comment]
Ciao
La potenza del laser è controllata esclusivamente dal valore S-Max nella finestra di impostazioni di import immagine.

![image](https://user-images.githubusercontent.com/8782035/98470577-df734400-21e6-11eb-9985-2cbcd245a1c7.png)

Il database materiali non è altro che una scorciatoia per richiamare valori pre-impostati di potenza e velocità secondo dati forniti dal produttore. Al momento Ortur è l'unico produttore che ha fornito dei dati per i suoi prodotti.
Se trovi delle combinazioni di velocità e potenza che vanno bene per la tua marcatrice, sui vari materiali, puoi aggiungerli tu stesso al database, basta aggiungere delle righe all'elenco.

**Comment by wiwan1971** [comment]
Grazie x l'informazione ciao

Il dom 8 nov 2020, 17:24 arkypita <notifications@github.com> ha scritto:
